### ISSUE TITLE
archinstall  cant start

### ISSUE THREAD

**Issue by q2333gh** [archlinux/archinstall#1994]
![IMG_20230807_224800](https://github.com/archlinux/archinstall/assets/32679742/97562310-71c0-4375-9cb9-b8d048c4aaca)
since I use bare metal install this. cannot export the error to text.

**Comment by bigo72** [comment]
Same issue here ... I'm going to use an old iso instead. Hoping they fix it soon

**Comment by q2333gh** [comment]
I tried use my familiar one Linux-distro : ubuntu-server. it worked. hah. just need to get things done and continue work with computer.

**Comment by 4ctiv** [comment]
I had the same issue on my pc.
 I tryed booting via usb and ventoy. 
The ventoy stick also had an encrypted partition on it. 
I also tryed to use the script on another usb which only has the arch iso flashed onto via etcher, this solved all the issues and the install is now running smoothly. 
Note: I used the same image on both sticks and verified the 256 checksums to be correct.
I think the encrypted partition via windows bitlocker was the problem did you also had simmilar partitions on time of the error ?
